flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/maintenance/api/RewriteDataFiles.java

```diff
@@ -170,6 +170,12 @@ public Builder maxFileGroupSizeBytes(long maxFileGroupSizeBytes) {
       return this;
     }
 
+    public Builder maxFilesToRewrite(Integer maxFilesToRewrite) {
```

Comment:
Should be `int`. I don't think you want to set it to `null`

Reply:
This is already null on the spark side of things and I chose this path to keep things constent. I can change it to Integer instead of `int` if you think that is the right way to go @pvary  ?

Reply:
I mean, it could be null by default. But setting in to `null` doesn't make any sense.
Please change the builder parameter to `int`

Reply:
Thank you . I changed the type to an int instead of Integer